### PR TITLE
Integration tests fix

### PR DESCRIPTION
**Pull request by botanical**
Wait for IngestGranule workflows to complete before deleting

### CODE REVIEW

**Review by Jkovarik — APPROVED** [review]

File: example/spec/serial/DiscoverGranulesHttpSpec.js [hunk 1]
```diff
@@ -183,15 +183,25 @@ describe('The Discover Granules workflow with http Protocol', () => {
...
       ingestGranuleWorkflowArn2 = queueGranulesOutput.payload.running[1];
       ingestGranuleWorkflowArn3 = queueGranulesOutput.payload.running[2];
-      console.log('\nwait for ingestGranuleWorkflow', ingestGranuleWorkflowArn1);
-      ingestGranuleExecutionStatus = await waitForCompletedExecution(ingestGranuleWorkflowArn1);
     });
 
-    it('executes successfully', () => {
+    it('executes successfully', async () => {
+      console.log('\nwait for ingestGranuleWorkflow', ingestGranuleWorkflowArn1);
+      ingestGranuleExecutionStatus = await waitForCompletedExecution(ingestGranuleWorkflowArn1);
```
Comment (Jkovarik) [inline example/spec/serial/DiscoverGranulesHttpSpec.js:194]: Why not wait for API on this execution as well?

**Review by Jkovarik — CHANGES_REQUESTED** [review]
Just one quick question

File: example/spec/serial/DiscoverGranulesHttpSpec.js (hunk 1 above)
Comment (Jkovarik) [reply example/spec/serial/DiscoverGranulesHttpSpec.js:194]: Or more importantly, this is already called in the after all: 

```
    await Promise.all(queueGranulesOutput.payload.running
      .map((arn) => waitForApiStatus(
        getExecution,
        {
          prefix: config.stackName,
          arn,
        },
        'completed'
      )));
```

Comment (botanical) [reply example/spec/serial/DiscoverGranulesHttpSpec.js:194]: Good point, this was just a copy paste. I can update this!

File: example/spec/parallel/discoverGranules/DiscoverGranulesS3SuccessSpec.js [hunk 2]
```diff
@@ -126,6 +126,11 @@ describe('The DiscoverGranules workflow', () => {
       }
     ));
 
+    await waitForApiStatus(
```
Comment (Jkovarik) [inline example/spec/parallel/discoverGranules/DiscoverGranulesS3SuccessSpec.js:129]: Should we be waiting for both executions?

**Review by Jkovarik — CHANGES_REQUESTED** [review]
One minor bit of feed back, else looks good!

File: example/spec/parallel/discoverGranules/DiscoverGranulesS3SuccessSpec.js (hunk 2 above)
Comment (botanical) [reply example/spec/parallel/discoverGranules/DiscoverGranulesS3SuccessSpec.js]: Yes [`ca5727d` (#2334)](https://github.com/nasa/cumulus/pull/2334/commits/ca5727df7dfd1c035245249a804b9eea5187be88)

**Review by Jkovarik — APPROVED** [review]
LGTM

**Pull request merged**